langchain4j-azure-open-ai/src/test/java/dev/langchain4j/model/azure/AzureOpenAiChatModelIT.java

```diff
@@ -278,7 +279,7 @@ void should_use_json_format(String deploymentName, String gptVersion) {
                 .apiKey(System.getenv("AZURE_OPENAI_KEY"))
                 .deploymentName(deploymentName)
                 .tokenizer(new AzureOpenAiTokenizer(gptVersion))
-                .responseFormat(new ChatCompletionsJsonResponseFormat())
+                .responseFormat(ResponseFormat.JSON)
```

Comment:
`ResponseFormat.JSON` does not work either, there is no `"response_format":{"type":"json_object"}` in the request.

Reply:
We need to improve this test to catch such things. I guess `GitHubModelsStreamingChatModelIT.should_use_json_format()` is more reliable. Returned content in this test is wrapped into ````json`, `GitHubModelsStreamingChatModelIT`'s way would catch it.

Reply:
Indeed, I just fixed it in my last commit, and added some unit test (not perfect) for the old method

Reply:
Thank you!